src/components/common/child-entry/child-entry.styles.tsx

```diff
@@ -0,0 +1,32 @@
+import { Color } from "@lifesg/react-design-system/color";
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    background: ${Color.Neutral[7]};
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    align-items: center;
+`;
+
+export const ChildStyles = styled.div`
+    padding: 1rem;
+    width: 100%;
+    flex-grow: 1;
+`;
+
+export const DeleteValidationButton = styled.div`
```

Comment:
ChildEntry is meant to be used beyond validation, it needs to support conditional rendering and prefilling. hence need to avoid mentioning "validation" here